regxmllib/CMakeLists.txt

```diff
@@ -44,3 +45,17 @@ endforeach( UNIT_TEST_PATH ${UNIT_TESTS} )
 #add_executable(AUIDTest src/test/cpp/com/sandflow/smpte/util/AUIDTest.cpp)
 #target_link_libraries(AUIDTest regxmllibc)
 #add_test(AUIDTest AUIDTest)
+
+file( GLOB_RECURSE HEADER_FILE_LIST ${CMAKE_SOURCE_DIR}/src/main/cpp/com/*.h )
+foreach(HEADER_FILE_PATH ${HEADER_FILE_LIST} )
```

Comment:
Could this be shortened to:

```
install(DIRECTORY src/ DESTINATION include/myproj
        FILES_MATCHING PATTERN "*.h")
```

See https://cmake.org/cmake/help/v3.5/command/install.html

Reply:
yes you're correct I will use the shortened version.

since all the .h files are located ${CMAKE_SOURCE_DIR}/src/main/cpp/com/sandflow can we use the below code instead?

install(DIRECTORY ${CMAKE_SOURCE_DIR}/src/main/cpp/com/sandflow  DESTINATION include
        FILES_MATCHING PATTERN "*.h")

if we use "src/" it will install the entire file structure even with directories that don't have header files such as src/test/java/com/sandflow/smpte/regxml and all the java directories. We don't want clutter in our installation.